src/components/review-form/review-form.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC, Fragment, ReactEventHandler, useState } from 'react';
+
+type TChangeHandler = ReactEventHandler<HTMLInputElement | HTMLTextAreaElement>;
+
+const ratings = [
+  { value: 5, title: 'perfect' },
+  { value: 4, title: 'good' },
+  { value: 3, title: 'not bad' },
+  { value: 2, title: 'badly' },
+  { value: 1, title: 'terribly' },
+];
+
+export const ReviewForm: FC = () => {
+  const [review, setReview] = useState({ rating: 0, review: '' });
+
+  const handleChange: TChangeHandler = (event) => {
```

Comment:
Обычная реализация все же, ChangeEvent из реакта
```suggestion
  const handleChange = (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
```